lib/std/os/linux.zig

```diff
@@ -3996,14 +3996,18 @@ pub const io_uring_probe_op = extern struct {
 pub const io_uring_probe = extern struct {
     /// last opcode supported
     last_op: IORING_OP,
-
     /// Number of io_uring_probe_op following
     ops_len: u8,
 
     resv: u16,
-    resv2: u32[3],
+    resv2: [3]u32,
+
+    /// To access all operations use `getOps`
+    first_op: io_uring_probe_op,
```

Comment:
This field makes the struct larger than the one defined in C.

Reply:
Ok, I fixed it.